src/renderer/components/test-utils/get-application-builder.tsx

```diff
@@ -703,6 +706,13 @@ export const getApplicationBuilder = () => {
       await startApplication({ shouldStartHidden: true });
     },
 
+    quit() {
```

Comment:
1. This isn't correct way to quit the application. This should use all same code than we use when application is quit in real life. Extract the function used in production and use it here.

2. `quit` shouldn't be part of all behavioural tests. There should be explicit behavioural tests for quitting the application and rest of them shouldn't know anything about it. That being said, `afterEach` requirement is not welcome.